x/xerrors/errors.go

```diff
@@ -23,3 +26,16 @@ func (e Errors) Error() string {
 
 	return strings.Join(s, "\n")
 }
+
+// SyncErrors is an error for tracing multiple errors safe to use in multiple goroutines.
+type SyncErrors struct {
```

Comment:
I prefer to have _Append_ and _AppendSync_ methods for _Error_ type. I think we don't quite need two types here.